### PR TITLE
fix (addon): #1013 only send tab data if there is an active tab

### LINKED ISSUE #1013: Intermittent failure: A coding exception was thrown and uncaught - TypeError: tabs.activeTab is null
I'm seeing this on my Mac fairly frequently when running `npm run test:jpm`

```
JPM [error] *************************
A coding exception was thrown and uncaught in a Task.


Full message: TypeError: tabs.activeTab is null
Full stack: handlePerformanceEvent@resource://gre/modules/commonjs/toolkit/loader.js -> resource://activity-streams/lib/TabTracker.js:120:5
_getEnhancedLinks@resource://gre/modules/commonjs/toolkit/loader.js -> resource://activity-streams/lib/PreviewProvider.js:226:5
getLinkMetadata@resource://gre/modules/commonjs/toolkit/loader.js -> resource://activity-streams/lib/PreviewProvider.js:214:23
RecommendationProvider.prototype.asyncSetRecommendedContent<@resource://gre/modules/commonjs/toolkit/loader.js -> resource://activity-streams/lib/RecommendationProvider.js:90:45
TaskImpl_run@resource://gre/modules/Task.jsm:319:40

*************************
```

### PR DESCRIPTION
**Pull request by rlr**
Should Fix #1013 and #1022

r?

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-stream/1031)

### CODE REVIEW

**Comment by rlr** [comment]
no fails! 💃

File: lib/TabTracker.js [hunk 1]
```diff
@@ -117,10 +117,12 @@ TabTracker.prototype = {
   handlePerformanceEvent(eventData, eventName, value) {
     let payload = Object.assign({}, eventData);
     payload.action = "activity_stream_performance";
-    payload.tab_id = tabs.activeTab.id;
     payload.event = eventName;
     payload.value = value;
-    this._setCommonProperties(payload, tabs.activeTab.url);
+    if (tabs.activeTab) {
```
Comment (ncloudioj) [inline lib/TabTracker.js]: I think we can just short circuit here if there is no activeTab property in tabs, cause it's pointless to send the telemetry in that case.

Comment (rlr) [inline lib/TabTracker.js]: ok, I can do that instead. I wasnt sure if it was useful or not

Comment (ncloudioj) [inline lib/TabTracker.js]: awesome! :8ball:

**Comment by ncloudioj** [comment]
r+ after fixing a nit.

**Pull request merged**